Bottom: d139533518ec366831cc17d52a2edd1e035adb4b
Top:    122c6a6fa8538ddc3b72a2843ce7b6e30ae00681
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-06-28 12:19:44 +0800

Refresh of vhost-introduce-new-ioctls-to

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 4abf8d1..25f2cc5 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1156,11 +1156,7 @@ static int vhost_iotlb_miss(struct vhost_virtqueue *vq, u64 iova, int access)
 	struct vhost_iotlb_msg *msg;
 	bool v2 = vhost_backend_has_feature(vq, VHOST_BACKEND_F_MSG_V2);
 
-<<<<<<< current
-	node = vhost_new_msg(vq, VHOST_IOTLB_MISS);
-=======
 	node = vhost_new_msg(vq, v2? VHOST_IOTLB_MSG_V2 : VHOST_IOTLB_MSG);
->>>>>>> patched
 	if (!node)
 		return -ENOMEM;
